mkosi/config.py

```diff
@@ -157,6 +157,8 @@ class OutputFormat(StrEnum):
     sysext    = enum.auto()
     tar       = enum.auto()
     uki       = enum.auto()
+    # TODO Do we want to support oci-dir or oci-tar, or both?
+    oci       = enum.auto()
```

Comment:
Go with oci_dir, since that's it and it leaves more open
```suggestion
    # TODO Do we want to support oci-dir or oci-tar, or both?
    oci       = enum.auto()
```

Reply:
I wanted to go with the nomenclature used by the containers tools which use [`oci` and `oci-archive`](https://github.com/containers/image/blob/278eaa3c1b629f0438e784cde703403c44898fc4/docs/containers-transports.5.md#ocipathreference)

Reply:
`docker image import` only seems to allow importing tars so supporting both might be a good idea (at least in the long run).

Reply:
> I wanted to go with the nomenclature used by the containers tools which use [oci and oci-archive](https://github.com/containers/image/blob/278eaa3c1b629f0438e784cde703403c44898fc4/docs/containers-transports.5.md#ocipathreference)

If that's the official nomenclature, then let's go with that. 

> docker image import only seems to allow importing tars so supporting both might be a good idea (at least in the long run).

Is docker still a thing? I thought all the cool kids these days used podman.

Reply:
> If that's the official nomenclature, then let's go with that.

Not by the [opencontainers initiative](https://github.com/opencontainers) itself as they do not specify how image names should be resolved but its the name used by the [containers projects](https://github.com/containers) which actually support working directly with local OCIs.

> Is docker still a thing?

Not on anything I touch but yes, still alive and kicking.